packages/shortest/src/ai/tools/index.ts

```diff
@@ -30,6 +30,42 @@ export const AITools = [
       required: ["action", "username", "password"],
     },
   },
+  {
+    name: "check_email",
+    description: "Fetch, decode and render received email in browser",
+    input_schema: {
+      type: "object",
+      properties: {
+        action: {
+          type: "string",
+          enum: ["check_email"],
+          description: "Render the received email in a new tab"
+        }
+      },
+      required: ["action", "email"]
+    }
+  },
```

Comment:
In this file we register our tools. We have created a new tool called "check_email". It let's AI know what we expect from it when calling this tool. The required field at the bottom asks AI to provide the email address as an ActionInput.